lib/class-wp-rest-menu-items-controller.php

```diff
@@ -260,13 +296,17 @@ protected function prepare_item_for_database( $request ) {
 			}
 		}
 
+		// Check if object id existing before saving.
```

Comment:
Nitpick: `// Check if object id exists before saving.`

Reply:
Can you give some more details here? @draganescu

Reply:
oh @spacedmonkey I wasn't clear: it's the spelling of the comment:

`// Check if object id exists before saving.`

instead of

`// Check if object id existing before saving.`

I told you, nitpick.

Reply:
Fixed in https://github.com/WP-API/menus-endpoints/pull/31/commits/cbc17732475ae99893696091debfa3d59dc41e18